addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1965,6 +1965,26 @@ options.registry.topMenuColor = options.Class.extend({
     },
 });
 
+options.registry.showMobile = options.Class.extend({
+    isTopOption: true,
+    selectClass: function (previewMode, widgetValue, params) {
+        this._super(...arguments);
+        this.renderTitle(widgetValue);
+    },
+    start: function (){
+        this._super(...arguments);
+        this.renderTitle(this.$target.hasClass('d-none d-md-flex'));
+    },
+    renderTitle: function (isHide){
+        if (isHide) {
+            this.$el.find('we-button').attr('title', "Visible on mobile");
+        }
+        else{
+            this.$el.find('we-button').attr('title', "Hidden on mobile");
+        }
+    }
```

Comment:
- If hidden you show "Visible on mobile" ? It should be the opposite 
    - it's just the way you look at it (I wanted to show the user what will happen if you click on it), but I changed my vision 😓 .
- Just calling it in the start method and selectClass does the job.

Reply:
> I wanted to show the user what will happen if you click on it

Then in that case, it should be "Show on mobile" instead of "Visible on mobile" (a verb :wink:), I am fine either way.